backend/editor/entries.py

```diff
@@ -206,7 +225,13 @@ def update_node_children(entry, new_children_ids):
     existing_ids = [record['child.id'] for record in get_current_transaction().run(query, ids=list(added_children))]
     to_create = added_children - set(existing_ids)
 
-    for child in to_create:
+    #Normalising new children node ID
+    normalised_added_children = set()
+    for child_node in to_create:
+        child_language_code, new_child_name = child_node.split(":",1)
+        normalised_added_children.add(normalizing(new_child_name,child_language_code))

```

Comment:
As create_node is used below and it already normalize id I don't think we need this (and this avoid duplication of code).

Instead I propose to build  normalised_added_children while creating nodes (see below)